src/main/java/com/yahoo/bullet/rest/query/QueryError.java

```diff
@@ -12,6 +12,8 @@
 
 @Getter
 public class QueryError {
+    public static final QueryError INVALID_QUERY = new QueryError("Failed to parse query.", "Please provide a valid query.");
+    public static final QueryError UNSUPPORTED_QUERY = new QueryError("Doesn't support window query for this endpoint", "Please provide a valid query without window.");
```

Comment:
How about "This particular REST endpoint does not support windowed queries", "Please provide a valid query without a window, or POST to a different REST endpoint."

Reply:
If you want to make it more verbose, we should say "This endpoint does not support windowed queries", "Please use the SSE or WS endpoints to submit queries with windows". 

Because you don't POST to the ws endpoint and the ws endpoint isn't really REST or HTTP based.

Reply:
Done.

Reply:
Sounds good, tho "WS" sounds a lot like "web service", maybe "Please use the SSE or Web Socket endpoints to submit queries with windows."

Reply:
👍